src/app/authorization/registration/registration.component.html

```diff
@@ -1,237 +1,232 @@
-<div class="container registration-conatiner">
-  <div class="card shadow">
-    <div class="card-body">
-      <form
-        [formGroup]="registrationForm"
-        (ngSubmit)="submitform(registrationForm.value)"
-        class="p-3"
-      >
-        <div class="row">
-          <div class="col text-center p-2 mb-3">
-            <span class="fs-3">User Registration</span>
+<div class="card shadow col-lg-4 col-md-4 col-sm-8 col-xs-8 mt-5">
```

Comment:
same here wrap conditions to ()

Reply:
I have Remove conditions only  for using col-sm-3 and mt-5